Modules/quote/Controller/QuotelistController.php

```diff
@@ -122,24 +122,42 @@ public function editexistinguserAction($id_space, $id) {
 
         $modelUser = new CoreUser();
         $users = $modelUser->getSpaceActiveUsersForSelect($id_space, "name");
+        $form->addHidden('id_space', $id_space);
         $form->addSelect('id_user', CoreTranslator::User($lang), $users["names"], $users["ids"], $info['id_user']);
 
+        $clientSelect['choices'] = [""];
+        $clientSelect['choicesid'] = [""];
+        $clientSelect['value'] = "";
+
         if ($id > 0) {
             $form->addText('date_open', QuoteTranslator::DateCreated($lang), false, CoreTranslator::dateFromEn($info['date_open'], $lang), 'disabled');
-            //$form->addHidden('date_open', CoreTranslator::dateFromEn($info['date_open'], $lang));
+            $form->addHidden('date_open', $info['date_open']);
+            $modelClientUser = new ClClientUser();
+            $userClients = $modelClientUser->getUserClientAccounts($info['id_user'], $id_space);
+            foreach($userClients as $client) {
+                array_push($clientSelect['choices'], $client['name']);
+                array_push($clientSelect['choicesid'], $client['id']);
+            }
+            $clientSelect['value'] = ($info['id_client'] != 0) ? $info['id_client'] : $userClients[0]['id'];
         } else {
             $form->addHidden('date_open', date('Y-m-d'));
         }
 
+        $form->addSelect('id_client', ClientsTranslator::Client($lang), $clientSelect['choices'], $clientSelect['choicesid'], $clientSelect['value']);
         $form->setButtonsWidth(2, 10);
         $form->setValidationButton(CoreTranslator::Save($lang), "quoteuser/" . $id_space . "/" . $id);
-
+        
         if ($form->check()) {
-
-            //echo "date open = " . $form->getParameter('date_open') . "<br/>";
-            //return;
-            $id = $modelQuote->set($id, $id_space, "", "", "", $form->getParameter('id_user'),
-                    $this->request->getParameterNoException('date_open')
+            $id = $modelQuote->set(
+                $id,
+                $id_space,
+                "",
+                "",
+                "",
+                "",
+                $form->getParameter('id_user'),
+                $form->getParameter('id_client'),
+                $this->request->getParameter('date_open')
```

Comment:
was $this->request->getParameterNoException('date_open')  ie not mandatory
you change to getParameter (must be present)

any reason?